packages/filesystem/src/node/nsfw-watcher/nsfw-options.ts

```diff
@@ -20,4 +20,7 @@ import * as nsfw from 'nsfw';
  * Inversify service identifier allowing extensions to override options passed to nsfw by the file watcher.
  */
 export const NsfwOptions = Symbol('NsfwOptions');
-export type NsfwOptions = nsfw.Options;
+export interface NsfwOptions {
+    nsfw: nsfw.Options;
+    entryPoint?: string;
```

Comment:
This is a breaking change that can be avoided by adding it's own binding for `entryPoint`: `NsfwServerEntryPoint` or something along those lines, wdyt?

Reply:
Better: put it as an optional value under `NsfwFileSystemWatcherServerOptions` ?